ios/RNModules/RNLogging/RNLogging/RNLoggingModule.m

```diff
@@ -62,12 +62,12 @@ + (BOOL)requiresMainQueueSetup {
 
 - (NSDictionary *)constantsToExport
 {
-    return @{ @"ANCILLARY_STEP_SEARCH_FORM" : @"searchForm",
-              @"ANCILLARY_STEP_RESULTS"     : @"results",
-              @"ANCILLARY_STEP_DETAILS"     : @"details",
-              @"ANCILLARY_STEP_PAYMENT"     : @"payment",
-              @"ANCILLARY_PROVIDER_BOOKINGCOM"     : @"booking.com",
-              @"ANCILLARY_PROVIDER_STAY22"     : @"stay22",
+    return @{ @"ANCILLARY_STEP_SEARCH_FORM"     : @"searchForm",
```

Comment:
Is there some `prettier` kind of formatter for Objective C?

Reply:
I think there is some formatting built in into Xcode